pvlib/iotools/tmy.py

```diff
@@ -25,7 +23,7 @@ def read_tmy3(filename=None, coerce_year=None, recolumn=True):
     ----------
     filename : None or string, default None
         If None, attempts to use a Tkinter file browser. A string can be
```

Comment:
Conversation here https://github.com/pvlib/pvlib-python/pull/677#discussion_r267812264 has me thinking that we should remove `tkinter` at the same time.

Reply:
Shall I remove it from `read_tmy2` as well?

Reply:
good idea